app/models/user.rb

```diff
@@ -81,6 +82,8 @@ class User < ApplicationRecord
   scope :active, -> { where(suspended_at: nil) }
   scope :suspended, -> { where.not(suspended_at: nil) }
 
+  enum email_notification: { unfilled: 0, yes: 1, no: 2}, _prefix: :email_notification
```

Comment:
since the default is `1`, do we ever get into a state where we will have an `unfilled` value? If so, should we just have `yes` or `no`?